### ISSUE TITLE
Exporting >40000 messages, freezes into gray box at the end.

### ISSUE THREAD

**Issue by marcellonovak** [prathercc/discrub-ext#144]
![image_2024-11-24_225008632](https://github.com/user-attachments/assets/f0c69ead-a370-4064-98c2-9a52066cdd40)
Works on smaller message sets, just not this for some reason.
Using opera GX

**Comment by shironaomi** [comment]
Yes, I've the same problem: https://github.com/prathercc/discrub-ext/issues/164
